runtime/src/main/cpp/Runtime.cpp

```diff
@@ -81,14 +82,15 @@ RuntimeState* initRuntime() {
   if (!result) return nullptr;
   result->memoryState = InitMemory();
   bool firstRuntime = atomicAdd(&aliveRuntimesCount, 1) == 1;
+  runtimeInitInProgress = true;
```

Comment:
Although this indeed fixes the bug, this code now looks much like requiring some refactoring: not having `::runtimeState` initialized during legitimate Kotlin code execution is somewhat confusing.